src/catkin_pkg/package_templates.py

```diff
@@ -41,6 +41,8 @@
 from catkin_pkg.package import Package
 from catkin_pkg.package import PACKAGE_MANIFEST_FILENAME
 from catkin_pkg.package import Person
+from catkin_pkg.cmake import get_metapackage_cmake_template_path
+from catkin_pkg.cmake import configure_file
```

Comment:
Oh, the packages as well, on the way.

Well, I meant the order of the function definitions (from where I took the order for importing before), first __g__`get_metapackage_cmake_template_path`, then __c__f__`configure_file` and finally __c__s__`configure_string`, but probably that's not PEP8, I haven't read it fully, so nevermind.